components/tidb_query/src/rpn_expr/impl_math.rs

```diff
@@ -96,6 +96,18 @@ impl Ceil for CeilDecToDec {
     }
 }
 
+pub struct CeilIntToDec;
+
+impl Ceil for CeilIntToDec {
+    type Input = Int;
+    type Output = Decimal;
+
+    #[inline]
+    fn ceil(_ctx: &mut EvalContext, arg: &Self::Input) -> Result<Option<Self::Output>> {
+        Ok(Some(Self::Output::from(*arg)))
```

Comment:
Please use `Decimal::from`, it's more clear.

Reply:
updated